apps/meteor/client/views/admin/users/UsersTable/UsersTableRow.tsx

```diff
@@ -53,39 +130,68 @@ const UsersTableRow = ({ user, onClick, mediaQuery, tab }: UsersTableRowProps):
 		>
 			<GenericTableCell withTruncatedText>
 				<Box display='flex' alignItems='center'>
-					{username && <UserAvatar size={mediaQuery ? 'x28' : 'x40'} username={username} etag={avatarETag} />}
-					<Box display='flex' mi={8} withTruncatedText>
-						<Box display='flex' flexDirection='column' alignSelf='center' withTruncatedText>
-							<Box fontScale='p2m' color='default' withTruncatedText>
-								<Box display='inline' mie='x8'>
-									<UserStatus status={status || Status.OFFLINE} />
-								</Box>
-								{name || username}
-							</Box>
-							{!mediaQuery && name && (
-								<Box fontScale='p2' color='hint' withTruncatedText>
-									{`@${username}`}
-								</Box>
-							)}
+					{username && <UserAvatar size={isMobile || isLaptop ? 'x28' : 'x40'} username={username} etag={avatarETag} />}
+					<Box display='flex' flexGrow={1} flexShrink={1} flexBasis='0%' alignSelf='center' alignItems='center' withTruncatedText>
+						<Box mi={8}>
+							<UserStatus status={status || Status.OFFLINE} />
+						</Box>
+						<Box fontScale='p2' withTruncatedText>
+							{name || username}
```

Comment:
Yes, but the problem here is that if I show any other column on mobile it will cause issues with the "Resend welcome email" due to the large width of it.